### PR TITLE
Add Dockerfile to run external-screen

### PR DESCRIPTION
**Pull request by schemen**
Allows for easy setup on a container host/cluster

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Stage1
+FROM node:lts as node
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm ci
```
Comment (Rycochet) [inline Dockerfile:8]: Should probably be -

```suggestion
RUN npm ci --production
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN npm install -g @angular/cli 
+
+COPY . .
+
+RUN ng build
+
+# Stage 2
+FROM nginx:1.13.12-alpine
```
Comment (Rycochet) [inline Dockerfile]: Very old version, any reason why not simply `FROM nginx:stable-alpine` ?

Comment (schemen) [reply Dockerfile]: Sorry, this was a mistake from copying another docker image I had! 
Will change of course.

Comment (Rycochet) [reply Dockerfile]: No worries, this is what community review is for - wondering if I'd missed something where I was using it :-P

File: Dockerfile (hunk 1 above)
Comment (schemen) [reply Dockerfile:8]: I'm afraid I'm no node expert, just dockerizing this by using the install instructions.

Adding the `--production` flag leads to a error in the `RUN ng build` section: 
```
An unhandled exception occurred: Cannot find module '@angular-devkit/build-angular/package.json'
Require stack:
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/architect/node/node-modules-architect-host.js
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/architect/node/index.js
- /usr/local/lib/node_modules/@angular/cli/models/architect-command.js
- /usr/local/lib/node_modules/@angular/cli/commands/build-impl.js
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/schematics/tools/index.js
- /usr/local/lib/node_modules/@angular/cli/utilities/json-schema.js
- /usr/local/lib/node_modules/@angular/cli/models/command-runner.js
- /usr/local/lib/node_modules/@angular/cli/lib/cli/index.js
- /usr/local/lib/node_modules/@angular/cli/lib/init.js
- /usr/local/lib/node_modules/@angular/cli/bin/ng
See "/tmp/ng-eIwsIt/angular-errors.log" for further details.
```
Any idea?

Comment (Rycochet) [reply Dockerfile:8]: Doh, my bad - forget that comment, this is the "build" stage - I totally missed it as used to CAPITAL_LETTER_NAMES for it lol

Comment (schemen) [reply Dockerfile:8]: Yeah, I try to make this image as tiny as possible as it's essentially a static site :)

File: Dockerfile (hunk 2 above)
Comment (schemen) [reply Dockerfile]: I have updated the nginx image to the stable image.

It seems that it's also working just fine, I have pushed the new docker image to docker hub.

**Pull request merged**